test/raygun_offline_test.js

```diff
@@ -17,18 +17,7 @@ test("offline message storage and sending", async function (t) {
     },
   });
   const raygunClient = testEnvironment.client;
-  const send = (e) =>
-    new Promise((resolve, reject) => {
-      raygunClient.send(e, null, (err, data) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(data);
-        }
-      });
-    });
-
-  await send(new Error("offline error"));
```

Comment:
this is no longer necessary, because now the `raygunClient.send` method is asynchronous